src/theme/components/global.ts

```diff
@@ -125,6 +125,10 @@ const labelLegendText = {
     color: "dark.ui.typography.heading",
   },
 };
+// Used in form inputs that require nuanced spacing.
+const labelLegendTextSpecialSpacing = {
+  marginBottom: "xxxs",
```

Comment:
Why not add this rule to `fieldset.ts` and `label.ts`? This seems like it'll be easier to miss if the update is in the component file than the style file.

Reply:
I was following the pattern of `labelLegendText` because I wanted a reusable object.

Reply:
Yes, but `labelLegendText` is used in `fieldset.ts` and `label.ts` and not in the `__css` prop for the components. So what's here is fine but if we can keep the update in the theme files, that'd be ideal.